client/card-readers/declarations.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module 'wcpay/settings/settings-section' {
+	interface SettingsSectionProps {
+		/* eslint-disable-next-line @typescript-eslint/naming-convention */
+		Description: () => JSX.Element;
```

Comment:
Can we follow naming conventions and use `Description`?

Reply:
Good point, I think I am going to migrate `SettingsSectionProps` also.

Reply:
@kalessil I went for a different approach here. We have a TS linter rule that enforces camelCase or snake_case for property names. I refactored the `SettingsSection` component to use `description` as the name of the property. It is worth mentioning that in the `SettingsSection` component I had to alias it to `Description`. This is because `<description />` throws a warning from React. Instead, I can alias it and use it like `<Description />`.

Please let me know your thoughts.